content/docs/guides/flyway.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Get started with Flyway and Neon
+subtitle: Learn how to manage schema changes in Neon with the Flyway command-line tool
+enableTableOfContents: true
+---
+
+Flyway is a database migration tool that facilitates version control for databases. It allows developers to manage and track changes to the database schema, ensuring that the database evolves consistently across different environments.
+
+This guide steps you through installing the Flyway command-line tool, configuring Flyway to connect to a Neon database, and running database migrations. The guide follows the setup described in the [Flyway command-line quickstart](https://documentation.red-gate.com/fd/quickstart-command-line-184127576.html).
+
+## Prerequisites
+
+- A Neon account. See [Sign up](/docs/get-started-with-neon/signing-up).
+- A Neon project. See [Create your first project](/docs/get-started-with-neon/setting-up-a-project).
+- A database. This guide uses the ready-to-use `neondb` database. You can create your own database if you like. See [Create a database](/docs/manage/databases#create-a-database) for instructions.
+
+## Download and extract Flyway
+
+1. Download the [Flyway command-line tool](https://documentation.red-gate.com/fd/command-line-184127404.html).
+
+2. Extract the Flyway files. For example:
+
+    ```bash
+    cd ~/Downloads
+    tar -xzvf flyway-commandline-9.22.3-linux-x64.tar.gz -C ~/
+    ```
+
+3. Open a command prompt to view the contents of your Flyway installation:
+
+    ```bash
+    cd ~/flyway-9.22.3
```

Comment:
FYI, version is now 10.0.1. Update? or doesn't matter

Reply:
changed to flyway-x.y.z